docs/src/ttrt.md

```diff
@@ -181,9 +185,11 @@ Note: It's required to be on a system with silicon and to have a runtime enabled
 
 ```bash
 ttrt query --help
+ttrt query --print
```

Comment:
Is this not just `query` with no arguments?

Reply:
it does store "state" which is not like an API design for sure, so for the next API related refactor, I'll return the actual system desc instead of storing it in the class instance (via functions rather than classes like we discussed)

Reply:
So `ttrt query` == `ttrt query --save-artifacts` now?

Reply:
ttrt query doesn't do anything from the cmd line POV. If you want to save artifacts, you have to explicitly provide the flag. wdyt, do you think this change is unnecessary? I wanted to keep save-artifacts consistent with all other API options, indifferent as to what to keep this as.

Reply:
added quiet option pursuant to offline conversation